src/cmd/tools/zarf.go

```diff
@@ -76,27 +89,36 @@ var updateCredsCmd = &cobra.Command{
 	Example: lang.CmdToolsUpdateCredsExample,
 	Aliases: []string{"uc"},
 	Args:    cobra.MaximumNArgs(1),
-	Run: func(cmd *cobra.Command, args []string) {
+	RunE: func(cmd *cobra.Command, args []string) error {
 		validKeys := []string{message.RegistryKey, message.GitKey, message.ArtifactKey, message.AgentKey}
 		if len(args) == 0 {
 			args = validKeys
 		} else {
 			if !slices.Contains(validKeys, args[0]) {
 				cmd.Help()
-				message.Fatalf(nil, lang.CmdToolsUpdateCredsInvalidServiceErr, message.RegistryKey, message.GitKey, message.ArtifactKey)
+				return fmt.Errorf("invalid service key specified, valid keys are: %s, %s, and %s", message.RegistryKey, message.GitKey, message.ArtifactKey)
 			}
 		}
 
+		timeoutCtx, cancel := context.WithTimeout(cmd.Context(), cluster.DefaultTimeout)
+		defer cancel()
+		c, err := cluster.NewClusterWithWait(timeoutCtx)
+		if err != nil {
+			return err
+		}
+
 		ctx := cmd.Context()
```

Comment:
Would it make sense to either move this `ctx` variable assignment to the top of this `RunE` function and reuse it, or remove it and use `cmd.Context()` directly?